python_modules/dagster/dagster_tests/execution_tests/test_execute_in_process.py

```diff
@@ -61,92 +43,6 @@ def emit_three():
     )
 
 
-def test_execute_solid_with_inputs():
```

Comment:
@dpeng817 it looks like we're losing a decent bit of test coverage here - is that intentional?

Reply:
I think any test coverage that we're losing is testing old, unused functionality of execute in process.

Reply:
It looks like `test_dynamic_output_solid` is exercising an `InProcessSolidResult.output_values` code path that isn't necessarily exercised elsewhere? Very possible I'm missing something.

Reply:
Hm I think you're right. I'll put up a PR to re-add a test around this.